pkg/templates/latesttag/template_test.go

```diff
@@ -38,9 +38,10 @@ func (s *ContainerImageTestSuite) addDeploymentWithContainerImage(name, containe
 }
 
 func (s *ContainerImageTestSuite) TestImproperContainerImage() {
+	//#nosec G101 -- This is a false positive
```

Comment:
Should we just disable this rule? It doesn't seem very applicable to kube-linter anyway, I suspect it will just give us false positives.